code/api/src/modules/product/model.js

```diff
@@ -22,4 +22,10 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.TEXT
     }
   })
+
+  Product.associate = function(models) {
+    Product.belongsToMany(models.Shipment, { through: models.ProductShipment })
```

Comment:
according to sequelize docs this is how you set up a many to many relationship, so hopefully it works when I start seeding and testing